java/src/main/c++/vw_learner_VWTopicLearner.cc

```diff
@@ -0,0 +1,31 @@
+#include "../../../../vowpalwabbit/parser.h"
+#include "../../../../vowpalwabbit/vw.h"
+#include "vw_errors.h"
+#include "vw_learner_VWTopicLearner.h"
+
+JNIEXPORT jfloatArray JNICALL Java_vw_learner_VWTopicLearner_predict(JNIEnv *env, jobject obj, jstring example_string, jboolean learn, jlong vwPtr) {
+    jfloatArray r = env->NewFloatArray(0);
+    try {
+        vw* vwInstance = (vw*)vwPtr;
+        const char *utf_string = env->GetStringUTFChars(example_string, NULL);
+        example *ec = VW::read_example(*vwInstance, utf_string);
+
+        vwInstance->l->predict(*ec);
+
+        r = env->NewFloatArray(vwInstance->lda);
```

Comment:
I was speaking with John and I think it would desirable to not mention any LDA code in this function.  If I understand this correctly vwInstance->lda is the number of topics.  This should be reproducible by using

```
size_t topics = ec->topic_predictions.size;
```

By doing this and replacing all `vwInstance->lda` with `topics` then if the float array return type is used somewhere else, John should change the name of the variable to something more generic than topic_predictions and then this function will be changed as well to reference that more generic variable name.